bot.go

```diff
@@ -45,6 +46,7 @@ type Bot struct {
 	Token   string
 	Updates chan Update
 	Poller  Poller
+	Proxy   string
```

Comment:
```go
// Proxy is a full address to redirect every request into.
```

Reply:
I'm not quite sure what to write here to make it better